app/build.gradle

```diff
@@ -219,6 +219,13 @@ dependencies {
     androidTestImplementation 'com.android.support.test.uiautomator:uiautomator-v18:2.1.3'
     androidTestImplementation 'com.squareup.okhttp3:mockwebserver:3.7.0'
     androidTestImplementation "android.arch.persistence.room:testing:$rootProject.architectureComponentsVersion"
+    androidTestImplementation 'com.android.support.test.espresso:espresso-contrib:' + rootProject.espressoVersion, {
```

Comment:
To sync with others, suggest to declare in this way => "com.android.support.test.espresso:espresso-contrib:$rootProject.espressoVersion"